vllm/model_executor/models/llama.py

```diff
@@ -378,11 +382,11 @@ def sample(
     def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
         stacked_params_mapping = [
             # (param_name, shard_name, shard_id)
-            ("qkv_proj", "q_proj", "q"),
-            ("qkv_proj", "k_proj", "k"),
-            ("qkv_proj", "v_proj", "v"),
-            ("gate_up_proj", "gate_proj", 0),
-            ("gate_up_proj", "up_proj", 1),
+            (".qkv_proj", ".q_proj", "q"),
```

Comment:
A quick question: Why adding dot here?

Reply:
Does Phi contain a different set of mappings?

Reply:
Does Phi have a different set of mappings? 

+ does this not break the existing llama executor?

Reply:
This ensures that we are replacing the full layer instead of a partial layer name.

Phi3 uses the vLLM layer naming convention (no replacement necessary), therefore we have a layer named _gate_up_proj_, without the dots, this layer would be incorrectly renamed to _gate_gate_up_proj_ due to line 389.

So the dot in _up_proj_ makes sure that we only change the name of the layer if the layer name starts with "up_proj" instead of containing "up_proj" anywhere in its name.

This change is compatible with Llama and Mistral models.

Reply:
Thanks for clarification! make sense.